services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/schema/JsonSchemaManagerImpl.java

```diff
@@ -225,6 +243,19 @@ public CreateSchemaResponse createJsonSchema(UserInfo user, CreateSchemaRequest
 		return response;
 	}
 	
+	private void sendUpdateNotifications(String schemaId) {
+		// Send update notifications to entities bound to the schema
+		Iterator<Long> objectIds = jsonSchemaDao.getObjectIdsBoundToSchemaIterator(schemaId);
+		while(objectIds.hasNext()) {
+			Long id = objectIds.next();
+			EntityType entityType = nodeDao.getNodeTypeById(KeyFactory.keyToString(id));
+			transactionalMessenger.sendMessageAfterCommit(id.toString(), ObjectType.ENTITY, ChangeType.UPDATE);
+			if (NodeUtils.isProjectOrFolder(entityType)) {
+				transactionalMessenger.sendMessageAfterCommit(id.toString(), ObjectType.ENTITY_CONTAINER, ChangeType.UPDATE);
+			}
+		}
+	}
```

Comment:
when(nodeDao.getNodeType(syn1).thenReturn(File);
1-> file
2-> folder
3-> project

3 calls to send with ObjectType.entity
2 call to send with ObjectType.container